package.json

```diff
@@ -25,7 +25,8 @@
     "standard-wdio": "wdio run ./tests/e2e/standard/wdio.conf.js",
     "default-wdio-mobile": "wdio run ./tests/e2e/default-mobile/wdio.conf.js",
     "local-images": "export VERSION=$(node ./scripts/build/get-version.js) && grunt build-service-images && grunt create-local-docker-compose-files",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "fresh-start": "npm ci && cd webapp && ng cache clean && cd .. && echo \"The fresh-start process has completed!\""
```

Comment:
I know you mentioned about intending to add this as `prepare`. 
Since you're attaching `ng cache clean` to `npm ci`, we could it as a `postinstall` script instead of an explicit command. 

This way: 
- we make sure we always clean cache on angular dependency changes
- less people will encounter the watches error because cache will be slim

What do you think?

Reply:
I understand the benefit, but I'm not sure if I did what you had in mind. Can you please have a look?